src/systems/systems/src/landing_gear/mod.rs

```diff
@@ -797,6 +898,10 @@ impl SimulationElement for LandingGearControlInterfaceUnit {
         );
 
         writer.write(&self.fault_ecam_id, self.status() != LgciuStatus::Ok);
+
+        writer.write(&self.discrete_word_1_id, f64::from(self.discrete_word_1()));
+        writer.write(&self.discrete_word_2_id, f64::from(self.discrete_word_2()));
+        writer.write(&self.discrete_word_3_id, f64::from(self.discrete_word_3()));
```

Comment:
**suggestion**: Add a `Writer` implementation for `Arinc429Word<f32>` which then calls the appropriate `From` implementation:

```rust
impl<T: Writer> Write<Arinc429Word<f32>> for T {
    fn convert(&mut self, value: Arinc429Word<f32>) -> f64 {
        value.into()
    }
}
```

```suggestion
        writer.write(&self.discrete_word_1_id, self.discrete_word_1());
        writer.write(&self.discrete_word_2_id, self.discrete_word_2());
        writer.write(&self.discrete_word_3_id, self.discrete_word_3());
```